src/DeviceListener.ts

```diff
@@ -324,7 +339,16 @@ export default class DeviceListener {
                 (await crypto.getDeviceVerificationStatus(cli.getSafeUserId(), cli.deviceId!))?.crossSigningVerified,
             );
 
-        const allSystemsReady = crossSigningReady && secretStorageReady && allCrossSigningSecretsCached;
+        const keyStorageOn = await this.getKeyBackupStatus();
```

Comment:
I notice `getKeyBackupStatus` doesn't check if we are *downloading* keys, only if we are *uploading* them. Is this correct behaviour? (I think maybe we need a *different* check and behaviour for that case? cf https://github.com/element-hq/element-web/issues/29872)

It should probably be called `keyBackupActive` and have a comment that it means "keys are being uploaded"

Reply:
Fixed in ff8e7ee1b7